### PR TITLE
Feature/published docs

### PR DESCRIPTION
**Pull request by timabrmsn**
Initial work to migrate dev portal content to mkdocs and publish Incydr SDK/CLI and API docs all in one place. 

Once we configure github pages for the repo, the docs should be visible to those who have access to the repo and will be "public" once we make the repo public.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -29,24 +31,33 @@ edit_uri: ""
...
-  - File Events: 'file_events.md'
-  - File Event Queries: 'file_event_queries.md'
-  - Legal Hold: 'legal_hold.md'
-  - Trusted Activites: 'trusted_activities.md'
-  - Users: 'users.md'
-  - User Risk Profiles: 'user_risk_profiles.md'
-  - Watchlists: 'watchlists.md'
-  - Models: 'models.md'
+  - API Reference: 'api-reference.md'
+  - SDK:
```
Comment (tora-kozic) [inline mkdocs.yml:35]: Are we going to be maintaining the SDK docs in multiple places?

**Comment by tora-kozic** [comment]
Are there any complications with hosting all of our docs in the same repo as our source code?

If we plan to rewrite the baldur/dev portal, will that also all be here?

**Comment by tora-kozic** [comment]
Gotcha.  FWIW I'm not crazy about the idea of hosting all of our API documentation out of the same repo as the SDK source code, but sounds like thats still up in the air 👍 
Are we hosting this in a feature branch at all?

**Comment by timabrmsn** [comment]
>Are we hosting this in a feature branch at all?

Not currently, I think it would be too much to try to maintain two separate versions of the docs (one just SDK and one dev portal + SDK) since the file layout is different.

**Review by tora-kozic — APPROVED** [review]

**Pull request merged**